simulator/risc_v/t/unit_test.cpp

```diff
@@ -215,6 +215,39 @@ TEST_CASE("RISCV RV64 orn")
     CHECK( instr.get_v_dst( 0) == 0x4000'0000'0000'10ff);
 }
 
+TEST_CASE ("RISCV sbinv32")
+{
+    CHECK( RISCVInstr<uint32>(  0x68e797b3).get_disasm() == "sbinv $a5, $a5, $a4");
+    std::vector<TestData<uint32>> cases {
+        TestData<uint32>( 0x1, 0, 0x0),
+        TestData<uint32>( 0x1, 32, 0x0),
+        TestData<uint32>( 0x0, 35, 0x8),
+        TestData<uint32>( 0xcf, 6, 0x8f),
+        TestData<uint32>( 0x8f, 6, 0xcf)
+    };
+    for (std::size_t i = 0; i < cases.size(); i++) {
+        INFO( "Iteration: " << i);
+        cases[i].make_test("sbinv");
+    }
+}
+
+TEST_CASE ("RISCV sbinv64")
+{
+    CHECK( RISCVInstr<uint32>(  0x68e797b3).get_disasm() == "sbinv $a5, $a5, $a4");
+    std::vector<TestData<uint64>> cases {
+        TestData<uint64>( 0x1, 0, 0x0),
+        TestData<uint64>( 0x1, 32, 0x100000001),
+        TestData<uint64>( 0x1, 64, 0x0),
+	TestData<uint64>( 0x0, 67, 0x8),
```

Comment:
We do not use tabs: https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#indentation